src/pages/Home.js

```diff
@@ -0,0 +1,141 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Home.css'
+import PokemonTile from "../components/pokemonTIle/PokemonTile";
+import FlipThroughButton from "../components/flipthroughbutton/FlipThroughButton";
+import LimitSelector from "../components/limitselector/LimitSelector";
+import pokemonLogo from '../assets/pokemonlogo.png'
+
+
+function Home(props) {
+
+    const ENDPOINT = 'https://pokeapi.co/api/v2/'
+    const [error, toggleError] = useState(false)
+    const [isLoading, toggleIsLoading] = useState(false)
+    // states to store the offset and limit for the request
+    const [pokemonLimit, setPokemonLimit] = useState(20)
+    const [pokemonOffset, setPokemonOffset] = useState(0)
+    const [reachedEnd, setReachedEnd] = useState('')
```

Comment:
`reachedEnd` doet vermoeden dat het om een boolean waarde gaat, maar dat is niet zo